tuplex/python/src/PythonContext.cc

```diff
@@ -232,14 +232,18 @@ namespace tuplex {
         // create new partition on driver
         auto driver = _context->getDriver();
 
+        std::vector<std::tuple<size_t, PyObject*>> fallbackRows;
```

Comment:
some of this code here seems quite repetitive. Maybe restructure in functions/structs?